File: project/sumin-onboarding/survey-service/domain/src/main/java/com/innercircle/survey/domain/repository/SurveyResponseRepository.java
package com.innercircle.survey.domain.repository;

import com.innercircle.survey.domain.entity.Survey;
import com.innercircle.survey.domain.entity.SurveyResponse;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface SurveyResponseRepository extends JpaRepository<SurveyResponse, UUID> {
}

File: project/kyungrok-onboarding/src/main/kotlin/kr/innercircle/onboarding/survey/repository/SurveyResponseRepository.kt
package kr.innercircle.onboarding.survey.repository

import kr.innercircle.onboarding.survey.domain.SurveyResponse

/**
 * packageName : kr.innercircle.onboarding.survey.repository
 * fileName    : SurveyResponseRepository
 * author      : ckr
 * date        : 25. 6. 21.
 * description :
 */
interface SurveyResponseRepository {
    fun save(surveyResponse: SurveyResponse): SurveyResponse
}